### ISSUE TITLE
CIBA spec: Exception for CIBA on id_token_hint

### ISSUE THREAD

**Issue by marcusalmgren** [OpenBanking-Brasil/specs-seguranca#407]
The document https://github.com/OpenBanking-Brasil/specs-seguranca/blob/main/open-banking-brasil-financial-api-1_ID3.md [says that ](https://github.com/OpenBanking-Brasil/specs-seguranca/blob/main/open-banking-brasil-financial-api-1_ID3.md#authorization-server)the Authorization server "must refuse authentication requests that include an id_token_hint", and that clause should perhaps be rephrased to exclude the CIBA backchannel request to avoid possible conflicting statements.

**Comment by guilhermedecampo** [comment]
That’s correct, this phrase conflicts with [5.2.2.1](https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-financial-api-1_ID3-ptbr.html#section-5.2.2.1), it's reasonable to remove the item 21 from the 5.2.2.
